cmd/disco/utils.go

```diff
@@ -19,15 +20,19 @@ import (
 	"github.com/vechain/thor/v2/log"
 )
 
-func initLogger(lvl int) {
+func initLogger(lvl int) *slog.LevelVar {
 	logLevel := log.FromLegacyLevel(lvl)
+	var level slog.LevelVar
+	level.Set(logLevel)
```

Comment:
Out of curiosity, do we want the admin endpoint in disco ? I wonder if that would make sense.

Reply:
Not sure about the answer to this. @libotony , @darrenvechain Do we need to change the log verbosity of disco at runtime?

Reply:
IMO it's not needed, but I don't feel too strongly either way